lib/convection/model/cloudfile.rb

```diff
@@ -2,6 +2,8 @@
 require_relative '../dsl/helpers'
 require_relative '../model/attributes'
 require_relative '../model/template'
+require 'Benchmark'
```

Comment:
Are we still using this? I saw in your terminal that you were doing "Diff took x milliseconds", but that logic does not appear to be present anymore.